src/main/java/org/example/lesson4/juma/homework/Calculator.java

```diff
@@ -0,0 +1,54 @@
+package org.example.lesson4.juma.homework;
+
+import java.util.Scanner;
+
+public class Calculator {
+
+
+
+    public double num1;
+    public double num2;
+    private String calculatorName;
+
+    public double add(double num1, double num2){
+
+        return num1 + num2;
+    }
+    public double add(double num1,double num2,double num3){
+
+
+        return num1 + num2 + num3;
+    }
+
+    public void compare(){
```

Comment:
compare():
This should compare the **two values entered from the constructor not from Scanner** and print which one is bigger